Testing-Notes.md

```diff
@@ -0,0 +1,42 @@
+# Testing Notes
+
+## `/login`
+- `"username"` should not be case sensitive. It should be normalised before it is added to the db. <br>
+**Example**: passing `{"username": "Pavol"}` should be normalised to `{"username": "pavol"}`.
+
+- `"username"` should not take non-alphabetical characters and return an error straight away. Caveat: there is something to be said for allowing numerical characters in the username, but this should be done after careful deliberation <br>
+**Example**: passing `{"username": "Pavol."}` currently returns `User does not exist`
```

Comment:
This is correct behavior as user `Pavol.` does not exist in the DB. Comment on using alphabetical chars only below